troposphere/static/js/components/Master.jsx

```diff
@@ -55,6 +57,15 @@ export default React.createClass({
         // The code below is only relevant to logged in users
         if (!context.hasLoggedInUser()) return;
 
+        if (Raven && window.SENTRY_DSN){
+            if (! Raven.isSetup()) {
```

Comment:
An alternative would be moving this into `setRavenContext` a rename of `loadRavenData`, which would assert that Raven is setup before setting up raven.